Sources/Purchasing/StoreKitAbstractions/StoreProduct.swift

```diff
@@ -94,8 +94,10 @@ public typealias SK2Product = StoreKit.Product
     // switflint:enable missing_docs
 }
 
+@_spi(Internal) extension StoreProduct: StoreProductType { }
```

Comment:
Because `StoreProductType` is now `@_spi(Internal) public` but `StoreProduct` is `public`, the more restrictive conformance of `StoreProduct` to it must be done separately (this is enforced by the compiler)